projects/bp-gallery/src/components/common/OverviewContainer.tsx

```diff
@@ -1,20 +1,43 @@
 import { IconProps, Tab, Tabs, Tooltip } from '@mui/material';
-import { ReactElement, useState } from 'react';
+import { ReactElement, useEffect, useState } from 'react';
+import { useStorage } from '../../hooks/context-hooks';
 
 export interface OverviewContainerTab {
   title: string;
   icon: ReactElement<IconProps>;
   content: ReactElement;
 }
 
+export enum OverviewContainerPosition {
+  START_VIEW = 'start',
+  DISCOVER_VIEW = 'discover',
+  ARCHIVE_VIEW = 'archives',
+}
+
 const OverviewContainer = ({
   tabs,
   defaultValue = 0,
```

Comment:
I know it doesn't come from this PR but from #504, but maybe you could rename `defaultValue` to something like `defaultTabIndex`?